VMware Ref Arch updates to bring back external services op mode as an option (#1358)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/disk-requirements.html.md

## Unsupported Mounted Disk Types

The following are **generally not supported** mounted disk types:

* NFS
* SMB/CIFS

> The storage device/service used must be high-speed in both I/O and connectivity and highly reliable to meet performance requirements. Device types in the supported list will usually meet these requirements. Many standard NAS and other device types, however, will not perform at the level required. So only use a NAS or other device type not in the supported list if it is certain it can accommodate these requirements. 

Reviewer: ```suggestion
Terraform Enterprise's storage device or service must be highly reliable and high-speed in both I/O and connectivity to meet performance requirements. Device types in the supported list will usually meet these requirements, but many standard NAS and other device types will not perform at the level required. Only use a NAS or other device type not in the supported list if you are certain it can accommodate these requirements. 
```
